sdk/monitor/Azure.Monitor.Query/tests/MonitorQueryTestEnvironment.cs

```diff
@@ -21,5 +23,18 @@ public class MonitorQueryTestEnvironment : TestEnvironment
         public string ResourceId => GetRecordedVariable("RESOURCE_ID");
         public string StorageAccountName => GetRecordedVariable("STORAGE_ACCOUNT_NAME");
         public string StorageAccountId => GetRecordedVariable("STORAGE_ID");
+
```

Comment:
Does this comment need to be added?

Reply:
no, we were just trying to see if adding this would work, but unfortunately the tests are still hung